jest.config.js

```diff
@@ -0,0 +1,8 @@
+// Sync object
```

Comment:
I've updated the module template with `jest.config.ts` — thoughts on using this? (You'll need `ts-node` to support this.)